notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def compare_motif_list(df_motifs_a, df_motifs_b, motif_scoring_metric=motif_scoring_KL_divergence, plot_motif_probs=False):
</pre><ul><li>Are <code>df_motifs_a</code> and <code>df_motifs_b</code> going to be a pandas Series ?</li><li>We should add a <code>Callable</code> type hint to <code>motif_scoring_metric</code> .</li><li>We should add a <code>bool</code> type hint to <code>plot_motif_probs</code> .</li><li>We should add the <code>torch.Tensor</code> type hint to the function output.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>